rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -6434,6 +6434,30 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 
 	void FSM(spu_opcode_t op)
 	{
+		// FSM following a comparison instruction
+		if (match_vr<s8[16], s16[8], s32[4]>(op.ra, [&](auto c, auto MP)
```

Comment:
It should match s64[2] too I believe. Even if it's unused.

Reply:
ok